README.md

```diff
@@ -1,4 +1,4 @@
-[![Version](https://img.shields.io/badge/version-7.0.1--beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
+[![Version](https://img.shields.io/badge/version-7.1.0--beta-orange.svg)](https://github.com/0xProject/0x-mesh/releases)
```

Comment:
Will we always add the `beta` tag to V2 releases?

Reply:
Yes.